rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/InfrastructureManager.java

```diff
@@ -237,12 +241,12 @@ public void recoverPersistedInfraVariables(Map<String, Serializable> persistedIn
      *
      * @return nodes whose registration status is deploying or lost.
      */
-    public ArrayList<RMDeployingNode> getDeployingNodes() {
+    public ArrayList<RMDeployingNode> getDeployingAndLostNodes() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The return type of this method should be an interface such as "List" rather than the implementation "ArrayList". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)